ui/component/viewers/videoViewer/internal/plugins/videojs-snapshot-button/plugin.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import videojs from 'video.js';
+import type { Player } from '../../videojs';
+
+const VERSION = '1.0.0';
+const defaultOptions = {};
+
+type Options = {
+  fileTitle: string,
+  poster: string,
```

Comment:
Didn't look too deep, but seems like `poster` is unused?

Reply:
Yup thanks, was trying something but didn't decide to use it in the end